packages/mux-video-react/src/index.tsx

```diff
@@ -33,6 +36,7 @@ const MuxVideo = React.forwardRef<HTMLVideoElement | undefined, Partial<Props>>(
       startTime,
       src: outerSrc,
       children,
+      autoPlay,
```

Comment:
Like in mux-audio-react, make sure that the prop doesn't get added to the `<video>` element.